buildutils/src/utils.ts

```diff
@@ -123,11 +127,15 @@ export function writeJSONFile(filePath: string, data: JSONObject): boolean {
   } catch (e) {
     // no-op
   }
-  if (!JSONExt.deepEqual(data, orig)) {
+
+  // If values do not match, overwrite file.
+  try {
+    assert.deepEqual(data, orig);
```

Comment:
This seems to now be deprecated in favor of assert.deepStrictEqual: https://nodejs.org/api/assert.html#assertdeepequalactual-expected-message

Reply:
I am using strict deep equal by importing it like this:
```typescript
const assert = require('assert').strict;
```

Reply:
Yes, I know you're using it. My point is that the name is an alias and deprecated, according to the docs. Minor point, I know, but thought I would point it out.

Reply:
I misunderstood the `.strict` import to mean it was not deprecated. I've updated to use `.deepStrictEquals`